pyacal/_control_systems/epics/pv.py

```diff
@@ -1,19 +1,98 @@
-"""Sirius PV class."""
+"""PV class."""
 
 import epics as _epics
 
-from ... import _get_connections_dict, _get_facility
+from ... import _get_facility
 
 
-class PV(_epics.pv.PV):
+class PV:
     """PV class."""
 
-    def __init__(self, devname, propty, **kwargs):
+    def __init__(
+        self,
+        devname,
+        propty,
+        auto_monitor=True,
+        connection_timeout=None
+    ):
         """."""
         self.devname = devname
         self.propty = propty
-        mapping = _get_facility().alias_map[devname]
-        pvname = mapping['cs_devname']
-        pvname += mapping['cs_propties'][propty]['name']
-        super().__init__(pvname, **kwargs)
-        _get_connections_dict()[(devname, propty)] = self
+        facil = _get_facility()
+        pvname = facil.get_attribute_from_aliases('cs_devname', devname)
+        pvname += facil.get_attribute_from_aliases(
+            f'cs_propties.{propty}.name', devname
+        )
+
+        self._pvo = _epics.PV(
```

Comment:
I think we can use `epics.get_pv` here

Reply:
Maybe! Tomorrow I'll check that. Thanks!

Reply:
@anacso17, I cannot use `epics.get_pv` here because I want to have independent control of the `auto_monitor` property of the PVs.